doc/For developers/Developer guide/Performance/Testing.md

```diff
@@ -0,0 +1,182 @@
+- 1 container `sender` running on node A;
+- 1 container `receiver` running on node B;
+- `sender` and `receiver` exchange TCP segments using `NTttcp` tool using command line options from _Test scenarios_ section
+
+
+```
++----------------------+                       +----------------------+
+|                      |                       |                      |
+|  +------------+      |                       |      +------------+  |
+|  |  VMSWITCH  |      |                       |      |  VMSWITCH  |  |
+|  |            +--+ +---------+       +---------+ +--+            |  |
+|  +------------+  | |         |       |         | |  +------------+  |
+|    |        |    | |  10 Gb  +-------+  10 Gb  | |       |      |   |
+|    |        |    +-+   NIC   +-------+   NIC   +-+       |      |   |
+|  +----+ +------+   |         |       |         |    +------+ +----+ |
+|  |    | |      |   +---------+       +---------+    |      | |    | |
+|  | OS | | CONT |     |                       |      | CONT | | OS | |
+|  |    | |      |     |                       |      |      | |    | |
+|  +----+ +------+     |                       |      +------+ +----+ |
+|                      |                       |                      |
++----------------------+                       +----------------------+
+```
+
+### Setup
+
+```powershell
+# System setup
+Install-WindowsFeature Containers
+Install-WindowsFeature NET-Framework-Features
+Install-WindowsFeature Hyper-V -IncludeManagementTools
+Restart-Computer
+Install-Module DockerMsftProvider -Force
+Install-Package Docker -ProviderName DockerMsftProvider -Force -RequiredVersion 17.06.2-ee-16
+Restart-Computer
+
+# Docker setup (on both VMs)
+Start-Service Docker
+docker image pull microsoft/windowsservercore:ltsc2016
+docker network create -d transparent --subnet=172.16.0.0/24 --gateway=172.16.0.254 -o com.docker.network.windowsshim.interface="Ethernet1" mynetwork
+
+# On sender
+docker run -id --rm --network mynetwork --ip=172.16.0.21 --name sender microsoft/windowsservercore:ltsc2016 powershell
+docker cp .\NTttcp.exe sender:C:\
+docker exec -it sender powershell
+
+# On receiver
+docker run -id --rm --network mynetwork --ip=172.16.0.22 --name receiver microsoft/windowsservercore:ltsc2016 powershell
+docker cp .\NTttcp.exe receiver:C:\
+docker exec -it receiver powershell
+
+# Testing (commands inside container)
+sender   > .\NTttcp.exe -s -m 1,*,172.16.0.22 -l 128k -t 15
+receiver > .\NTttcp.exe -r -m 1,*,172.16.0.22 -rb 2M -t 15
+```
+
+
+### Results
+
+Across multiple runs of baseline scenario, the following average figures are observed:
+
+- on `sender` container:
+
+| Metric | Result |
+|---|---|
+| Throughput (MB/s) | 199.086 |
+| Retransmits | 41.9 |
+| Errors | 0 |
+| Avg. CPU % | 15.843 |
+
+- on `receiver` container:
+
+| Metric | Result |
+|---|---|
+| Throughput (MB/s) | 199.087 |
+| Retransmits | 0 |
+| Errors | 0 |
+| Avg. CPU % | 42.278 |
```

Comment:
We need to add discussion section.

Is this the expected result? Is something faster/slower than expected? Does Retransmits tell us something? etc

Also, how about moving results to a separate .md file and linking to it here? (can be in the same subdir)